From 8e86fdb3a7dd4bf13683ef9c1c737f46aed7c2e5 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:04:35 -0400
Subject: [usb] host: Remove superfluous name casts

Message-id: <1410887463-20674-204-git-send-email-dzickus@redhat.com>
Patchwork-id: 95251
O-Subject: [RHEL7 PATCH 203/591] usb: host: Remove superfluous name casts
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit a458677db6a928535a7186248983d1e192142e83
Author: Geert Uytterhoeven <geert@linux-m68k.org>
Date:   Tue Nov 12 20:07:25 2013 +0100

    usb: host: Remove superfluous name casts

    device_driver.name is "const char *"

    Signed-off-by: Geert Uytterhoeven <geert@linux-m68k.org>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/host/imx21-hcd.c b/drivers/usb/host/imx21-hcd.c
index 5c3fe46..46d0d3b 100644
--- a/drivers/usb/host/imx21-hcd.c
+++ b/drivers/usb/host/imx21-hcd.c
@@ -1930,7 +1930,7 @@ failed_request_mem:
 
 static struct platform_driver imx21_hcd_driver = {
  .driver = {
-     .name = (char *)hcd_name,
+     .name = hcd_name,
      },
  .probe = imx21_probe,
  .remove = imx21_remove,
diff --git a/drivers/usb/host/isp116x-hcd.c b/drivers/usb/host/isp116x-hcd.c
index b64e661..4d48ba0 100644
--- a/drivers/usb/host/isp116x-hcd.c
+++ b/drivers/usb/host/isp116x-hcd.c
@@ -1705,7 +1705,7 @@ static struct platform_driver isp116x_driver = {
  .suspend = isp116x_suspend,
  .resume = isp116x_resume,
  .driver = {
-  .name = (char *)hcd_name,
+  .name = hcd_name,
   .owner = THIS_MODULE,
  },
 };
diff --git a/drivers/usb/host/isp1362-hcd.c b/drivers/usb/host/isp1362-hcd.c
index c15463b..64cc31c 100644
--- a/drivers/usb/host/isp1362-hcd.c
+++ b/drivers/usb/host/isp1362-hcd.c
@@ -2829,7 +2829,7 @@ static struct platform_driver isp1362_driver = {
  .suspend = isp1362_suspend,
  .resume = isp1362_resume,
  .driver = {
-  .name = (char *)hcd_name,
+  .name = hcd_name,
   .owner = THIS_MODULE,
  },
 };
diff --git a/drivers/usb/host/r8a66597-hcd.c b/drivers/usb/host/r8a66597-hcd.c
index 6404c36..ec01087 100644
--- a/drivers/usb/host/r8a66597-hcd.c
+++ b/drivers/usb/host/r8a66597-hcd.c
@@ -2534,7 +2534,7 @@ static struct platform_driver r8a66597_driver = {
  .probe = r8a66597_probe,
  .remove = r8a66597_remove,
  .driver  = {
-  .name = (char *) hcd_name,
+  .name = hcd_name,
   .owner = THIS_MODULE,
   .pm = R8A66597_DEV_PM_OPS,
  },
diff --git a/drivers/usb/host/u132-hcd.c b/drivers/usb/host/u132-hcd.c
index 5c124bf..a32fd33 100644
--- a/drivers/usb/host/u132-hcd.c
+++ b/drivers/usb/host/u132-hcd.c
@@ -3217,7 +3217,7 @@ static struct platform_driver u132_platform_driver = {
  .suspend = u132_suspend,
  .resume = u132_resume,
  .driver = {
-     .name = (char *)hcd_name,
+     .name = hcd_name,
      .owner = THIS_MODULE,
      },
 };
-- 
1.7.1